content/languages/java/junit.md

```diff
@@ -2,3 +2,61 @@
 title: JUnit
 kind: reference
 ---
+
+You can use JUnit [4](https://junit.org/junit4/) or [5](https://junit.org/junit5/) in Codewars. 5 offers nesting, pretty name display and multiple test classes.
```

Comment:
Technically, multiple test classes are possible with JUnit 4 on Codewars too. It is clunky because of how test snippet is handled, but I remember I managed to use multiple test classes by nesting them inside of a top-level class.

Reply:
I'd be happy to see that. Feel free to PR!

Reply:
BTW do we even need to mention JUnit 4 in Codewars docs? It cannot be removed because there's many kata using it, but why not deprecate it and provide docs only for JUnit 5, in a similar way Javascript Codewars framework was available but discouraged and not extensively documented?

Reply:
I thought about that but wasn't sure I wasn't overreaching. If that's the general sentiment in the community (you're more in it than I), I'm all for stripping out the JUnit 4 stuff.